Resources/Computing/BOINCComputingElement.py

```diff
@@ -85,14 +84,14 @@ def submitJob( self, executableFile, proxy = None, numberOfJobs = 1 ):
       wrapperContent = """#!/bin/bash
 /usr/bin/env python << EOF
 # Wrapper script for executable and proxy
-import os, tempfile, sys, base64, bz2, shutil
+import os, tempfile, sys, base64, bz2, shutil, stat
```

Comment:
To follow on what @sposs and @acasajus  were saying this morning : https://www.python.org/dev/peps/pep-0008/#imports